src/utils/types.ts

```diff
@@ -14,3 +14,7 @@ export type TTextColors =
     | 'success'
     | 'warning'
     | 'white';
+
+export type TOrderIdsMap = {
```

Comment:
can we move this to the other types.ts file ?

Reply:
done 🙏